src/main/resources/assets/createframed/lang/ru-ru.json

```
@@ -0,0 +1,217 @@
+{
+  "itemGroup.createframed.create_framed": "Create: Framed",
```

Not a big deal, but this will need to be translated unless there isn't a fitting alternative.

---

"Create: Обрамление" or "Create: Обрамлённость" can fit. I prefer the first one.
```suggestion
  "itemGroup.createframed.create_framed": "Create: Обрамление",
```